### PR TITLE
Add exclude filter

### PR DESCRIPTION
**Pull request by luisesteves**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,6 +24,7 @@ Just invoke *logme* in your Fastfile action. Logme will return the commit messag
 | `to_revision` | End revision commit | false |
 | `from_revision` | From revision commit | false |
 | `message_regexp_filters` | Regexp filters like. ie '^MDM-\|^CTS:' | true |
+| `message_regexp_exclude_filters` | Regexp filters like. ie '^MDM-TST\|^CTS:UI' | true |
```
Comment (p4checo) [inline README.md:27]: Isn't this functionality achieved with a single regex filter? 🤔 

Additionally, I think that having two regex filters may create confusion on the caller as to how they interact. For instance, is the include filter run first and then the exclude, or vice versa?

Comment (luisesteves) [reply README.md:27]: @p4checo 
It doesn't matter who runs first (I think!)
["MDM-cenas", "NOJIRA - UI", "MDM-coisas", "batatas"].select{|c| c[/^MDM-|^NOJIRA/]!=nil}.select{|c| c[/^NOJIRA.*UI/] == nil}
 => ["MDM-cenas", "MDM-coisas"]
["MDM-cenas", "NOJIRA - UI", "MDM-coisas", "batatas"].select{|c| c[/^NOJIRA.*UI/] == nil}.select{|c| c[/^MDM-|^NOJIRA/]!=nil}
 => ["MDM-cenas", "MDM-coisas"]

Having in the same regexp, couldn't be very complex?

Comment (p4checo) [reply README.md:27]: This regex is simply a filter right? So it filters commits that should appear on the change log.

Why do we need two filters to filter the same thing?

Regular expressions allow negation of a pattern, so in the same regex you can have *including* and *excluding* patterns. In your example, the regex could be something like: `^MDM-|^NOJIRA-(?!.*UI)` (haven't tested it though 😇 )

**Comment by luisesteves** [comment]
That's it!

**Pull request closed without merging**